draft-ietf-httpbis-priority.md

```diff
@@ -125,9 +125,11 @@ a standardized and extensible format of priority information. {{header-field}}
 defines the Priority HTTP header field, a protocol-version-independent and
 end-to-end priority signal. Clients can use this header to signal priority to
 servers in order to specify the precedence of HTTP responses. Similarly, servers
-behind an intermediary can use it to signal priority to the intermediary.
+behind an intermediary can use it to signal priority to the intermediary. After
+a client sends a request, it may be beneficial for it to change the priority of
+the response, {{reprioritization}} describes this reprioritization.
 {{h2-update-frame}} and {{h3-update-frame}} define version-specific frames that
-carry priority parameters, which clients can use for reprioritization.
+carry priority parameters for reprioritization.
```

Comment:
How about merging the last two sentences to something like: _After sending a request, a client can change the priority of the response (see {{reprioritization}}), by sending HTTP-version-specific frames defined in {{frame}}._

I think we can omit the word "reprioritization" in the introduction, and that we do not need to state that the frames carry priority parameters, when we do not explicitly say that for the header field.

Reply:
I like that, added in https://github.com/httpwg/http-extensions/pull/1812/commits/0a8eeeacb4b25efe35a98afab5c94393a8cae66a